File: 1264-maximum-number-of-words-you-can-type/maximum-number-of-words-you-can-type.py
class Solution:
    def canBeTypedWords(self, text: str, brokenLetters: str) -> int:
        words = text.split()
        res = 0
        for word in words:
            if any([c in word for c in brokenLetters]):
                continue
            else:
                res += 1
        return res

File: 1264-maximum-number-of-words-you-can-type/maximum-number-of-words-you-can-type.cpp
class Solution {
public:
    int canBeTypedWords(string text, string brokenLetters) {
        int res = 0;
        unordered_set<char> broken(brokenLetters.begin(), brokenLetters.end());
        stringstream words(text);
        string word;
        while (words >> word){
            bool canType = true;
            for (char c : word){
                if (broken.count(c)){
                    canType = false;
                    break;
                }
            }
            if(canType) res++;
        }
        return res;
    }
};